src/components/SelectInputWithInfoLink.tsx

```diff
@@ -0,0 +1,41 @@
+import { FC, ChangeEvent, SelectHTMLAttributes } from 'react';
+import { InfoCircleWithTooltip } from './InfoCircleWithTooltip';
+
+interface SelectInputWithInfoLinkProps extends SelectHTMLAttributes<HTMLSelectElement> {
```

Comment:
Any reason why some of the default props are here? I think we can get rid of the non-custom ones